app/code/core/Maho/Captcha/etc/system.xml

```diff
@@ -12,8 +12,7 @@
                     <fields>
                         <enabled translate="label">
                             <label>Enable Captcha</label>
-                            <frontend_type>select</frontend_type>
-                            <source_model>adminhtml/system_config_source_yesno</source_model>
+                            <frontend_type>boolean</frontend_type>
```

Comment:
i also fixed a couple of system_config_source_yesno that I've seen just by chance

Reply:
Thank you!